lib/pages/create_post.dart

```diff
@@ -191,27 +230,34 @@ class CreatePostPage extends HookWidget {
     // TODO: use lazy autocomplete
     final communitiesDropdown = InputDecorator(
```

Comment:
hide communities dropdown when `_isEdit == true`

Reply:
Should instance drop-down be hidden too then?

Reply:
nah, instance can stary